opta/commands/config.py

```diff
@@ -0,0 +1,73 @@
+from typing import Optional
+
+import click
+from click_didyoumean import DYMGroup
+
+from opta.core.aws import AWS
+from opta.core.azure import Azure
+from opta.core.cloud_client import CloudClient
+from opta.core.gcp import GCP
+from opta.utils import logger
+
+
+@click.group(cls=DYMGroup)
+def config() -> None:
+    pass
+
+
+cloud_option = click.option(
+    "-c",
+    "--cloud",
+    help="The cloud provider to use",
+    required=True,
+    type=click.Choice(["aws", "google", "azurerm"]),
+)
+
+name_option = click.option(
+    "-n", "--name", help="The name of the configuration", required=False
+)
+
+environment_option = click.option(
+    "-e", "--environment", help="The name of the environment", required=False
+)
+
```

Comment:
let's stick to MVP, `opta config view --cloud aws/google/azurerm` 
I don't think we really had come up to a point where we thought the other options enough to implement them.